src/display/editor/annotation_editor_layer.js

```diff
@@ -577,6 +577,16 @@ class AnnotationEditorLayer {
    * @param {PointerEvent} event
    */
   pointerup(event) {
+    const isMac = KeyboardManager.platform.isMac;
+    if (event.button !== 0 || (event.ctrlKey && isMac)) {
+      // Don't create an editor on right click.
+      return;
+    }
+
+    if (event.button !== 0) {
+      return;
+    }
+
```

Comment:
Isn't this block completely redundant, given that you already return early above when this condition is satisfied?